src/wiki/development/build-instructions.md

```diff
@@ -166,36 +166,56 @@ We recommend using a build workflow based on MSYS2, as it's the easiest way to g
-2. Now we can prepare the build itself: Run `cmake -Bbuild -DCMAKE_INSTALL_PREFIX=install -DENABLE_LTO=ON -DLauncher_QT_VERSION_MAJOR=6 -DCMAKE_OBJDUMP=/mingw64/bin/objdump.exe -G Ninja`. These options will copy the final build to `C:\msys64\home\<your username>\PrismLauncher\install` after the build.
+1. `cd` into the folder you cloned Prism Launcher to. Put quotes around the path.
+2. Now we can prepare the build itself: Run `cmake -Bbuild -DCMAKE_INSTALL_PREFIX=install -DENABLE_LTO=ON -DLauncher_QT_VERSION_MAJOR=6 -DCMAKE_OBJDUMP=/mingw64/bin/objdump.exe -G Ninja`. These options will copy the final build to `install` after the build.
 
-   - NOTE: If you want to build using Qt 5, then remove the `-DLauncher_QT_VERSION_MAJOR=6` parameter
+   - If you want to build using Qt 5, then remove the `-DLauncher_QT_VERSION_MAJOR=6` parameter
+   - If you want to use [ccache](#ccache) to speed up recompilations, add the parameter `-DCMAKE_CXX_COMPILER_LAUNCHER=ccache`
 
 3. Now you need to run the build itself: Run `cmake --build build`.
 4. Now, wait for it to compile. This could take some time, so hopefully it compiles properly.
 5. Run the command `cmake --install build`, and it should install Prism Launcher to whatever the `-DCMAKE_INSTALL_PREFIX` was.
 6. If you don't want Prism Launcher to store its data in `%APPDATA%`, run `cmake --install build --component portable` after the install process
-7. When building on Qt 5, whenever compiling, the OpenSSL DLLs aren't put into the directory to where Prism Launcher installs which are necessary in that case, meaning that you cannot log in. The best way to fix this, is just to do `cp /(msystem))/bin/libcrypto-1_1.dll /(msystem))/bin/libssl-1_1.dll install`. This should copy the required OpenSSL DLLs to log in. When building on Qt 6 this is not necessary because it can use schannel, the Windows tls library.
+7. When building on Qt 5, whenever compiling, the OpenSSL DLLs aren't put into the directory to where Prism Launcher installs which are necessary in that case, meaning that you cannot log in. The best way to fix this, is just to do `cp /(msystem)/bin/libcrypto-1_1.dll /(msystem)/bin/libssl-1_1.dll install`. This should copy the required OpenSSL DLLs to log in. When building on Qt 6 this is not necessary because it can use schannel, the Windows tls library.
 
-   - Replace *(msystem)* with the msystem you're using (e.g. on *MSYS2 MinGW CLANG x86* clang32). On 64 bit msystems like *MSYS2 MinGW CLANG x64*, you have to add a -x64 to the dlls.
+   - Replace `(msystem)` with the msystem you're using (e.g. clang64). On 64-bit msystems, like *MSYS2 CLANG64*, you have to add `-x64` to the dlls.
+
+### Using Qt Creator (optional)
+
+1. Install Qt Creator within MSYS2 using `pacboy -S qt-creator:p`
+
+   - NOTE: If you install Qt Creator within Windows itself, Qt Creator will not be able to find anything installed in MSYS2.
+
+2. (Optional) Create a shortcut to `C:\msys64\(msystem).exe qtcreator`
```

Comment:
Interestingly, running `qtcreator.exe` directly also prevents Qt Creator from being able to find the compiler, so I just tell the user to run `qtcreator` from MSYS.

Reply:
I actually added this info into the note above. c: